db/migrate/20181027014323_add_applicable_modules.rb

```diff
@@ -0,0 +1,14 @@
+class AddApplicableModules < ActiveRecord::Migration[5.2]
+  def change
+    create_table "katello_content_facet_applicable_module_streams" do |t|
+      t.references 'content_facet', :null => false, :index => { :name => :katello_cfams_cf_idx }
+      t.references 'module_stream', :null => false, :index => { :name => :katello_cfams_ms_idx }
+    end
```

Comment:
I think you're missing foreign keys on these?

Reply:
addressed. Thanks!